diff --git a/mm/huge_memory.c b/mm/huge_memory.c
index 8847f84da129..0177b2fb50e0 100644
--- a/mm/huge_memory.c
+++ b/mm/huge_memory.c
@@ -2294,7 +2294,7 @@ static void unmap_page(struct page *page)
 	if (PageAnon(page))
 		ttu_flags |= TTU_SPLIT_FREEZE;
 
-	unmap_success = try_to_unmap(page, ttu_flags);
+	unmap_success = try_to_unmap(page, ttu_flags, NULL);
 	VM_BUG_ON_PAGE(!unmap_success, page);
 }
 
